### DISCUSSION TITLE
Cannot create personal access token

### DISCUSSION THREAD

**Discussion by dglav** [community/community#50777] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I cannot create a new personal access token (new format nor classic format).
When I click the button nothing happens, and an error message is posted in the terminal.
I was able to create a token last week, but today I am unable.

Error message:
```
html-safe-nonce.ts:41 Uncaught (in promise) Error: missing X-HTML-Safe nonce for HTTP 200
    at l (html-safe-nonce.ts:41:11)
    at createHTML (turbo.ts:7:5)
    at PageSnapshot.fromResponse (turbo.es2017-esm.js:1641:51)
    at turbo.es2017-esm.js:1856:61
    at Visit.render (turbo.es2017-esm.js:2043:15)
```

https://user-images.githubusercontent.com/34477242/227107258-a43c6c11-b5db-4a6d-b039-398a5ac14b80.mov

**Comment by xshi3x** [comment]
same issue

**Reply by arka2010** [reply]
same here.

**Reply by xshi3x** [reply]
I found that not only the token, but also all html-safe-nonce.ts can not be used.
I think this issue should be upgraded. **Needs to be fixed immediately.**

**Comment by katastic** [comment]
First time I go to use this interface in a year and I catch the exact hour it breaks.

**Comment by markhc** [comment]
Same here on Google Chrome.

Works fine on Firefox for me. You guys can try that as a workaround in the meantime while GH works to fix it.

**Comment by hpsin** [comment, marked as answer]
Hey all, 

We rolled out a security update for the GitHub.com web UI, but it broke quite a few things (2FA enablement, PATs creation, several other pages too).  It's been rolled back, so everything should work once again.

**Comment by dglav** [comment]
I verified that I can now create a new PAT. Thank you for the fix.

**Comment by eg-sidra** [comment]
I was trying to create a personal access token for git integration with jira , but after submitting generate token i am not able to get any token
<img width="795" alt="image" src="https://github.com/user-attachments/assets/ddc88d3a-4629-496c-8e5b-4170a1d82ba8" />
once i submit i am redirected to this page 
<img width="892" alt="image" src="https://github.com/user-attachments/assets/19d00fa8-8718-4679-9ff6-e726dd439c41" />

**Reply by Polari** [reply]
I had the same issue as above with Generate new token dumping me back to the previous menu, but got it to work by selecting 90 days instead of a longer custom expiration time

**Reply by martinjotpunkt** [reply]
@hpsin I can verify that the problem persists and that @Polari's workaround works.

**Comment by ktarbet** [comment]
I was *not* able to create personal access token with a custom expiration date.   @Polari 's workaound still works.
